CMakeLists.txt

```diff
@@ -1,8 +1,30 @@
-cmake_minimum_required(VERSION 3.13)
-PROJECT (dejavu_cpp_port)
+#./CMakeLists.txt
+cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+cmake_policy(VERSION 3.2)
+
+project(dejavu_cpp_port
+    VERSION "1.0.0"
+    DESCRIPTION "C++ port of dejavu lib"
+)
 find_package(OpenCV REQUIRED)
 
-set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR}/bin)
-add_executable(fingerprint fingerprint.cpp fingerprint.h)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
If you change that also update the README which indicates C++14

Reply:
yes